api/s2n.h

```diff
@@ -787,11 +810,13 @@ S2N_API extern int s2n_config_set_cert_chain_and_key_defaults(struct s2n_config
 
 /**
  * Adds to the trust store from a CA file or directory containing trusted certificates.
- * To completely override those locations, call s2n_config_wipe_trust_store() before calling
- * this function. 
  *
- * @note The trust store will be initialized with the common locations for the host
- * operating system by default.
+ * If default system certificates should not be present in the trust store in addition to the
```

Comment:
This paragraph is kind of passive and hard to follow. It might be clearer if you start with the problem you're trying to address and then talk about how to fix it using our APIs.